cmd/git_utils.go

```diff
@@ -209,9 +209,7 @@ func RunGitGetLastCommit(URL string, config *RootCommandConfig) (CommitInfo, err
 
 	projectDir, err := getProjectDir(config)
 	if err != nil {
-		if _, ok := err.(*NotAnAppsodyProject); ok {
-			// ignore this, we don't care it it is not an appsody project here
-		} else {
+		if _, ok := err.(*NotAnAppsodyProject); !ok {
```

Comment:
Just a suggestion, we could check that it's `AnAppsodyProject` inside of the double `No` with `NotAn` and `!ok`

Reply:
Although having the double negative is confusing, I think it makes more sense to keep, since we are checking to see if the error given by `getProjectDir` is `The current directory is not a valid appsody project. Run appsody init <stack> to create one. Run appsody list to see the available stacks.`